metrics/Gauge.md

```diff
@@ -0,0 +1,99 @@
+# Gauge Overview
+A `Gauge` is used to record aggregated metrics that can go up and down. Typical examples of gauges would be the number of jobs/entries in a queue,  number of threads in a running state or current memory usage etc.
+
+The `Gauge` values can be negative. This document describes the key types and the overall behavior of API.
+
+## Gauge API
+
+The value that is published for gauges is an instantaneous measurement of an `int64` or `double` value. This API is useful when you want to manually increase and decrease values as per service requirements.
+
+The following general operations MUST be provided by the API:
```

Comment:
Nit: do you need a line break here to make the bulleted list below be formatted right in Markdown?